docs/getting-started.md

```diff
@@ -68,6 +67,16 @@ The server is accessible via web console at:
     https://192.168.42.10:8443
 ```
 
+Now set your `PATH` to include the version of the `oc` tool bundled with Minishift.  You may already have `oc` installed separately, but this approach releases you from this requirement and additionally ensures that the versions between client and server are aligned.
+
+```console
+$ eval $(minishift oc-env)
+$ which oc
+~/.minishift/cache/oc/v3.10.0/linux/oc
+```
+
+Note that you will have to execute this `eval` command for every command shell you're using as its effects are not global.  Alternatively, you may install the `oc` tool system-wide and put it on the `PATH`, but you will need to ensure version compatibility.  You may take that approach [here](https://docs.openshift.org/latest/minishift/getting-started/quickstart.html#starting-minishift).
```

Comment:
Still don't know why we should add this.. shouldn't we keep this simple (using just `minishift start`?)

Reply:
Am sorry that I am just seeing this notification now, @cdrage.  The reasoning here is that `minishift start` is not the only command a user may execute.  I may use another shell to issue `oc` commands and I need to have the `oc` being used as both available and the correct version.